proposal/pachyderm.md

```diff
@@ -0,0 +1,246 @@
+          name: odhpachyderm-deployer
+  ```
+- KfDef for Pachyderm with Other Storage
+  ~~~
+    apiVersion: kfdef.apps.kubeflow.org/v1
+    kind: KfDef
+    metadata:
+      name: opendatahub
+      namespace: opendatahub
+    spec:
+      applications:
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: odh-common
+          name: odh-common
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: odh-dashboard
+          name: odh-dashboard
+        - kustomizeConfig:
+            parameters:
+              - name: s3_endpoint_url
+                value: s3.odh.com
+            repoRef:
+              name: manifests
+              path: jupyterhub/jupyterhub
+          name: jupyterhub
+        - kustomizeConfig:
+            overlays:
+              - additional
+            repoRef:
+              name: manifests
+              path: jupyterhub/notebook-images
+          name: notebook-images
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: ceph/object-storage/scc
+          name: ceph-nano-scc
+        - kustomizeConfig:
+            repoRef:
+              name: manifests
+              path: ceph/object-storage/nano
+          name: ceph-nano
+        - kustomizeConfig:
+            parameters:
+              - name: namespace
+                value: openshift-operators
+              - name: pachyderm_version
+                value: 0.0.8
+            repoRef:
+              name: manifests
+              path: odhpachyderm/operator
+          name: odhpachyderm-operator
+        - kustomizeConfig:
+            parameters:
+            - name: storage_secret
+              value: pachyderm-aws-secret
+            repoRef:
+              name: manifests
+              path: odhpachyderm/deployer
+          name: odhpachyderm-deployer  
+  ~~~
+
+**Upstream Pachyderm Components**
+- [Pachd](https://catalog.redhat.com/software/containers/pachyderm/pachd/617faaf02066d11ba7762a0e) (Go)
+- [Worker](https://catalog.redhat.com/software/containers/pachyderm/worker/617fab4dbe25a74c00922c12?tag=2.0.0-rc.1.rh.1&architecture=amd64&push_date=1635771274000&container-tabs=overview) (Go)
+- Postgresql
+- [PgBouncer](https://catalog.redhat.com/software/containers/pachyderm/pgbouncer/6168322f5cfcf7adc247b7ad)
+- [etcd](https://catalog.redhat.com/software/containers/pachyderm/etcd/618a8b52b876130c07c06f93)
+
+
+**Pachyderm Operator**
+- [Certified CatalogSource](https://catalog.redhat.com/software/containers/pachyderm/pachyderm-operator/61823a50dd607bfc82e65e14)
+- [Marketplace CatalogSource](https://marketplace.redhat.com/en-us/products/pachyderm)
+
+
+**Authentication**
+Community-Edition Pachyderm does not provide authentication features. However, Enterprise-Edition Pachyderm does provide authentication features and the possibility to plug into auth services such as Keycloak.
```

Comment:
This is a small nitpick, my concern is that this line opens the door to other open source components with a paid version using ODH to advertise features that are not available in the open source/community versions

Reply:
To me, it is proposal information for ODH integration. This does not have much meaning. In order to get accepted by the community, ISV should try to explain its applications in detail. Detail information should be better.
@nakfour What do you think about it?

Reply:
@Jooho I agree with Landon, ODH is an open source community and we should not be advertising non open source components

Reply:
@nakfour @LaVLaS So deleting the enterprise part would be ok? like the following:
~~~
Community-Edition Pachyderm does not provide authentication features.
~~~
If you agree with this sentence, I am going to update it.

Reply:
That looks good to me. It can be updated later if authentication support makes it in to the community edition

Reply:
Updated